packages/preprocess/src/process-queries.cjs

```diff
@@ -247,6 +227,27 @@ const createDefaultProps = function (filename, componentDevelopmentMode, duckdbQ
             }
         });
 
+		import { browser, dev } from "$app/environment";
```

Comment:
Why was this moved outside `queryDeclarations`?

Reply:
Because we want it to always be available; I had to put a dummy query on my page with the sql console so that the query deps would actually load onto the page